SOURCES.toml

```
@@ -502,14 +647,27 @@ path = "points.json"
 
 [[resources]] # Path: political-contributions.json
 path = "political-contributions.json"
-description = """Summary financial information on contributions to candidates for U.S. elections. An updated version of this datset is available from the \"all candidates\" files (in pipe-delimited format) on the bulk data download page of the U.S. Federal Election Commission, or, alternatively, via OpenFEC. Information on each of the 25 columns is available from the [FEC All Candidates File Description](https://www.fec.gov/campaign-finance-data/all-candidates-file-description/). The sample dataset in `political-contributions.json` contains 58 records with dates from 2015.
-FEC data is subject to the commission's:
+description = """Summary financial information on contributions to candidates for U.S. 
+elections. An updated version of this datset is available from the "all candidates" files 
+(in pipe-delimited format) on the bulk data download page of the U.S. Federal Election 
+Commission, or, alternatively, via OpenFEC. Information on each of the 25 columns is 
+available from the [FEC All Candidates File Description](https://www.fec.gov/
+campaign-finance-data/all-candidates-file-description/). The sample dataset in 
+`political-contributions.json` contains 58 records with dates from 2015.
 
+FEC data is subject to the commission's:
 - [Sale or Use Policy](https://www.fec.gov/updates/sale-or-use-contributor-information/)
 - [Privacy and Security Policy](https://www.fec.gov/about/privacy-and-security-policy/)
 - [Acceptable Use Policy](https://github.com/fecgov/FEC/blob/master/ACCEPTABLE-USE-POLICY.md)
+
 Additionally, the FEC's Github [repository](https://github.com/fecgov/FEC) states:
-> This project is in the public domain within the United States, and we waive worldwide copyright and related rights through [CC0 universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication. Read more on our license page. A few restrictions limit the way you can use FEC data. For example, you can't use contributor lists for commercial purposes or to solicit donations. Learn more on [FEC.gov](https://www.fec.gov/)."""
+> This project is in the public domain within the United States, and we waive worldwide 
+> copyright and related rights through [CC0 universal public domain](https://
+> creativecommons.org/publicdomain/zero/1.0/) dedication. Read more on our license page. 
```

@dsmedia you can exceed any line limit if it would break a link btw 

```suggestion
> copyright and related rights through [CC0 universal public domain](https://creativecommons.org/publicdomain/zero/1.0/)
> dedication. Read more on our license page.
```

Apologies for not mentioning that before. 
Thinking ahead, this would be a tricky one to fix since it has a `> ` break inside the link